cluster.tf

```diff
@@ -34,6 +34,19 @@ resource "aws_eks_cluster" "this" {
   ]
 }
 
+resource "aws_security_group_rule" "cluster_private_access" {
+  count       = var.create_eks && var.manage_aws_auth && var.cluster_endpoint_private_access && var.cluster_endpoint_public_access == false ? 1 : 0
+  type        = "ingress"
+  from_port   = 443
+  to_port     = 443
+  protocol    = "tcp"
+  cidr_blocks = [var.cluster_endpoint_public_access]
```

Comment:
```suggestion
  cidr_blocks = [var. cluster_endpoint_private_access_cidrs]
```

Reply:
Yup, cidrs is already a list though.

Reply:
True. I edited my suggestion to remove the brackets.

Reply:
fixed